report/src/main/kotlin/com/hartwig/actin/report/pdf/tables/trial/EligibleTrialGenerator.kt

```diff
@@ -228,6 +252,18 @@ class EligibleTrialGenerator(
             } else null
         }
 
+        private fun filterCohortsLateTrialPhase(cohorts: List<InterpretedCohort>): List<InterpretedCohort> {
+            return cohorts.filter { it.phase in TRIAL_LATE_PHASES }
+        }
+
+        private fun filterCohortsEarlyTrialPhase(cohorts: List<InterpretedCohort>): List<InterpretedCohort> {
+            return cohorts.filter { it !in filterCohortsLateTrialPhase(cohorts) }
```

Comment:
Much more efficient to use the same phase list so you don't have to filter the cohort list each time:
```suggestion
            return cohorts.filterNot { it.phase in TRIAL_LATE_PHASES }
```